src/pybind/mgr/dashboard/frontend/src/app/shared/components/card/card.component.html

```diff
@@ -1,7 +1,18 @@
-<div class="card shadow-sm flex-fill">
-  <h4 class="card-title mt-4 ms-4 mb-0">
-    {{ cardTitle }}
+<div class="card flex-fill"
+     [ngClass]="{'removeBorder': removeBorder}">
```

Comment:
you could use the bootstrap css rather than creating an extra css here
```suggestion
     [ngClass]="{'border-0': removeBorder}">
```

Reply:
Done